src/PyF/Internal/Meta.hs

```diff
@@ -0,0 +1,229 @@
+{-# LANGUAGE PackageImports #-}
```

Comment:
This is the module I did not wanted to write. I'm really looking forward for the end of "tree that grows" which is supposed to give us that for free.

However I'm surprised that it ends up as a really short module actually.